### DISCUSSION TITLE
MSG: "Your ComfyUI is too OUTDATED!!!"

### DISCUSSION THREAD

**Discussion by ghost** [comfyanonymous/ComfyUI#3927] [Q&A] [answered]
I just downloaded an reinstalled (manual install) ComfyUI, today. Now i got the message "Your ComfyUI is too OUTDATED!!!" in the manager. When trying to update "Failed to update ComfyUI." (Update ComfyUI, ComfyUI update fail: The installed ComfyUI does not have a Git repository.)
Versions: ComfyUI: Unknown[-] (1900-01-01)
Manager: V2.43

I LOVE COMFY! Thank you soooo much, @comfyanonymous !!!

**Comment by ltdrdata** [comment]
How did you install the ComfyUI?

**Reply by ghost** [reply]
Manually according to the "Manual Install (Windows, Linux)" section. As I did multiple times before.

**Reply by ltdrdata** [reply, marked as answer]
I am meaning what command did you use for install?
Your symptoms suggest that you have neither installed ComfyUI using git clone nor installed the portable version.
You should not use download this (Download zip).

![image](https://github.com/comfyanonymous/ComfyUI/assets/128333288/ec034dc1-fb64-4389-97e1-7199dc15dc6f)

**Reply by ghost** [reply]
Oh man, I am such a dumb ass... Sorry for wasting your time!

Thank you so much for your immediate answer and your hugh effort you're putting in to this project!!!

**Reply by wibur0620** [reply]
> I am meaning what command did you use for install? Your symptoms suggest that you have neither installed ComfyUI using git clone nor installed the portable version. You should not use download this (Download zip).
> 
> ![image](https://private-user-images.githubusercontent.com/128333288/345008497-ec034dc1-fb64-4389-97e1-7199dc15dc6f.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzMyOTk2MzYsIm5iZiI6MTczMzI5OTMzNiwicGF0aCI6Ii8xMjgzMzMyODgvMzQ1MDA4NDk3LWVjMDM0ZGMxLWZiNjQtNDM4OS05N2UxLTcxOTlkYzE1ZGM2Zi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIwNFQwODAyMTZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xNDJhMjdmOTMzY2JmZWU1MWU3NTVhMzhmNWJlZTJjZDVhZWYwYTE0NDQ5NzZjYTc2OTIwMjQ3NGZmNzliNjA5JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Z7Qfythdjmyc7LOE6mGu6Cu6krQY6kEyLBATo6CVlv0)

The desktop version of ComfyUI also has this issue.Since the previous version is too old,I couldn't find an option to upgrade ComfyUI in the desktop management(only"Update All Nodes"and"Get Updates"options are available).I clicked on both options,but in the end,it still prompted that the version is too outdated,which left me unsure of how to update the ComfyUI desktop end.Eventually,I uninstalled it and cleared the corresponding folders,then went to download the new ComfyUI desktop installation package,which is now version 0.3.23.However,the manager still prompts that the version is outdated.
![IMG20241204161137](https://github.com/user-attachments/assets/2759373c-36d1-4adb-bf29-41f572fb474c)
![IMG20241204161122](https://github.com/user-attachments/assets/4316d133-da81-4c4f-b470-85a00a5cb40a)

**Reply by ltdrdata** [reply]
For the desktop version, try updating it to the latest version.
There was a patch to ComfyUI-Manager related to that issue.

**Comment by Dod1** [comment]
Getting this same error with the new desktop version

**Reply by ltdrdata** [reply]
For the desktop version, try updating it to the latest version.
There was a patch to ComfyUI-Manager related to that issue.
